### PR TITLE
Correct near surface wind scaling

### PR DESCRIPTION
**Pull request by danholdaway**
## Description

This PR will add support for computing the near surface wind scaling variable correctly using the changes in UFO

## Dependencies

- [x] waiting on https://github.com/JCSDA-internal/ufo/pull/3050

## Impact

N/A

### CODE REVIEW

File: src/swell/configuration/jedi/interfaces/geos_atmosphere/observations/sfcship.yaml [hunk 1]
```diff
@@ -16,18 +16,22 @@ obs operator:
...
    - name: VertInterp
-     observation alias file: '{{experiment_root}}/{{experiment_id}}/configuration/jedi/interfaces/{{model_component}}/observations/obsop_name_map.yaml'
-     apply near surface wind scaling: true
      variables:
-     - name: airTemperature
-     - name: virtualTemperature
+     - name: windEastward
+     - name: windNorthward
+     # Use height vertical coordinate first
+     vertical coordinate: geopotential_height
```
Comment (gmao-jjin3) [inline src/swell/configuration/jedi/interfaces/geos_atmosphere/observations/sfcship.yaml:23]: It is "geometric_height" for surface and surface_ship winds:
https://github.com/GEOS-ESM/GEOSana_GridComp/blob/develop/GEOSaana_GridComp/GSI_GridComp/setupw.f90#L661

File: src/swell/configuration/jedi/interfaces/geos_atmosphere/observations/sfc.yaml [hunk 2]
```diff
@@ -16,16 +16,21 @@ obs operator:
...
 
    # Wind fields are handled using wind reduction factor applied to the lowest level.
-   - name: Product
-     geovals to scale hofx by: wind_reduction_factor_at_10m
+   - name: VertInterp
      variables:
      - name: windEastward
      - name: windNorthward
+     # Use height vertical coordinate first
+     vertical coordinate: geopotential_height
```
Comment (gmao-jjin3) [inline src/swell/configuration/jedi/interfaces/geos_atmosphere/observations/sfc.yaml:24]: It is "geometric_height" for surface and surface_ship winds:
https://github.com/GEOS-ESM/GEOSana_GridComp/blob/develop/GEOSaana_GridComp/GSI_GridComp/setupw.f90#L661

File: src/swell/configuration/jedi/interfaces/geos_atmosphere/observations/scatwind.yaml [hunk 3]
```diff
@@ -11,11 +11,30 @@ obs space:
...
 
 obs operator:
-  name: Product
+
+  name: VertInterp
   observation alias file: '{{experiment_root}}/{{experiment_id}}/configuration/jedi/interfaces/{{model_component}}/observations/obsop_name_map.yaml'
-  geovals to scale hofx by: wind_reduction_factor_at_10m
+
+  # Use height vertical coordinate first
+  vertical coordinate: geopotential_height
```
Comment (gmao-jjin3) [inline src/swell/configuration/jedi/interfaces/geos_atmosphere/observations/scatwind.yaml:19]: It is "geometric_height" for surface and surface_ship winds:
https://github.com/GEOS-ESM/GEOSana_GridComp/blob/develop/GEOSaana_GridComp/GSI_GridComp/setupw.f90#L661

**Review by gmao-jjin3 — APPROVED** [review]
All look good. GSI uses geometric_height for scatter wind, sfc and sfcship. I will have a test tomorrow.

File: src/swell/configuration/jedi/interfaces/geos_atmosphere/observations/sfcship.yaml (hunk 1 above)
Comment (danholdaway) [reply src/swell/configuration/jedi/interfaces/geos_atmosphere/observations/sfcship.yaml:23]: Perhaps that explains why the match for surface winds is not as exact as the others. But it's confusing by scatwind and sfc get a perfect match.

Comment (danholdaway) [reply src/swell/configuration/jedi/interfaces/geos_atmosphere/observations/sfcship.yaml:23]: What gets done for the temperature variable in the surface types?

**Comment by danholdaway** [comment]
I'm going to merge this but acknowledge that we should covert surface types to geometric height instead of geopotential height. The tests all look good so I think it's OK for now. We need the PRs in fv3-jedi to be able to provide geometric height. I believe Emily was going to work on this.

**Pull request merged**